packages/util-endpoints/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./EndpointError";
+export * from "./lib/";
```

Comment:
lib doesn't have to be exported, as it would be used only by endpoint resolver which would be in the same package.
```suggestion
```